scripts/compute_taxonomy_breakdown.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+#-----------------------------------------------------------------------------
+# Copyright (c) 2016--, Evguenia Kopylova.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+# Compute number of unique taxonomies and their counts
+
+import sys
+
+
+def get_genome_paths(scores_average_fp, scores_repophlan_fp):
+    """Return taxonomy strings and their counts.
+    """
+    genomes = []
+    # Get genome IDs
+    with open(scores_average_fp) as scores_average_f:
+        next(scores_average_f)
+        for line in scores_average_f:
+            line = line.strip().split('\t')
+            genome_id = line[0]
+            if genome_id not in genomes:
+                genomes.append(genome_id)
+            else:
+                raise ValueError("Duplicate genome IDs %s" % genome_id)
+    # Get taxonomies based on used genome IDs
+    taxonomies = {}
+    genomes_without_taxonomy = []
+    with open(scores_repophlan_fp) as scores_repophlan_f:
+        # header
+        line = scores_repophlan_f.readline()
+        line = line.strip().split('\t')
+        tax_idx = line.index('taxonomy')
+        for line in scores_repophlan_f:
+            line = line.strip().split('\t')
+            genome_id = line[0]
+            # only want tax_ids for genomes passing quality filter
+            if genome_id in genomes:
+                taxonomy = line[tax_idx]
+                if (('k__Bacteria' not in taxonomy) and
```

Comment:
it might be cleaner to define an exclude set, and test the length of a set intersection with taxonomy